src/main/java/org/web3j/protocol/core/personal/JsonRpc2_ExtendedWeb3j.java

```diff
@@ -0,0 +1,119 @@
+package org.web3j.protocol.core.personal;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.core.JsonRpc2_0Web3j;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.personal.methods.response.EcRecovered;
+import org.web3j.protocol.core.personal.methods.response.ImportedRawKey;
+import org.web3j.protocol.core.personal.methods.response.LockedAccount;
+import org.web3j.protocol.core.personal.methods.response.NewAccount;
+import org.web3j.protocol.core.personal.methods.response.Signed;
+import org.web3j.protocol.core.personal.methods.response.Transaction;
+import org.web3j.protocol.core.personal.methods.response.UnlockedAccount;
+import org.web3j.protocol.parity.methods.response.PersonalListAccounts;
+
+public class JsonRpc2_ExtendedWeb3j extends JsonRpc2_0Web3j
+implements Personal {
+
+	public JsonRpc2_ExtendedWeb3j(Web3jService web3jService) {
+		super(web3jService);
+	}
+
+	@Override
+	public Request<?, ImportedRawKey> personalImportRawKey(String hexStringNoPrefix, String passphrase){
+		return new Request<>(	
+				"personal_importRawKey",
```

Comment:
Indentation should be 8 characters under the return statement, not 16. Please ensure this is consistent in all files in the PR.